### PR TITLE
CMakeLists.txt: Install helper tools to CMAKE_INSTALL_BINDIR.

### PR DESCRIPTION
**Pull request by gerasiov**
Signed-off-by: Alexander GQ Gerasiov <gq@cs.msu.su>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -781,26 +781,31 @@ IF(CPUINFO_SUPPORTED_PLATFORM AND CPUINFO_BUILD_TOOLS)
...
   CPUINFO_TARGET_RUNTIME_LIBRARY(cache-info)
   TARGET_LINK_LIBRARIES(cache-info PRIVATE cpuinfo)
+  INSTALL(TARGETS cache-info RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
 
   IF(CMAKE_SYSTEM_NAME MATCHES "^(Android|Linux)$" AND CMAKE_SYSTEM_PROCESSOR MATCHES "^(armv[5-8].*|aarch64)$")
     ADD_EXECUTABLE(auxv-dump tools/auxv-dump.c)
     CPUINFO_TARGET_ENABLE_C99(auxv-dump)
     CPUINFO_TARGET_RUNTIME_LIBRARY(auxv-dump)
     TARGET_LINK_LIBRARIES(auxv-dump PRIVATE ${CMAKE_DL_LIBS} cpuinfo)
+    INSTALL(TARGETS auxv-dump RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```
Comment (Maratyszcza) [inline CMakeLists.txt]: I don't think its a good idea to install this binary, as it is just a development tool

File: CMakeLists.txt [hunk 2]
```diff
@@ -781,26 +781,31 @@ IF(CPUINFO_SUPPORTED_PLATFORM AND CPUINFO_BUILD_TOOLS)
...
 
     ADD_EXECUTABLE(cpuinfo-dump tools/cpuinfo-dump.c)
     CPUINFO_TARGET_ENABLE_C99(cpuinfo-dump)
     CPUINFO_TARGET_RUNTIME_LIBRARY(cpuinfo-dump)
+    INSTALL(TARGETS cpuinfo-dump RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```
Comment (Maratyszcza) [inline CMakeLists.txt]: Same here

File: CMakeLists.txt (hunk 1 above)
Comment (gerasiov) [reply CMakeLists.txt]: @cdluminate

Could you comment on this?

File: CMakeLists.txt (hunk 2 above)
Comment (gerasiov) [reply CMakeLists.txt]: @cdluminate

**Comment by cdluminate** [comment]
@gerasiov I think the two binaries should not be installed after reading the code. `auxv-dump` calls `getauxval` from libc and dumps the result. `cpuinfo-dump` simply does the same thing as `$ cat /proc/cpuinfo`.

**Comment by Maratyszcza** [comment]
@cdluminate LGTM, but I'm not longer at Facebook, someone else is to merge it. cc @hlu1, @harouwu, @Yangqing

**Pull request merged**